src/main/java/tqs/example/impostor/repository/Admin.java

```diff
@@ -0,0 +1,65 @@
+package tqs.example.impostor.repository;
+
+import jakarta.persistence.*;
+
+import java.util.Objects;
+
+@Entity
+@Table(name = "ADMIN")
+public class Admin {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "userName")
+    private String userName;
+
+    @Column(name = "password")
+    private String password;
+
+    public Admin() {
+    }
+
+    public Admin(String userName, String password) {
+        this.userName = userName;
+        this.password = password;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+        Admin admin = (Admin) o;
+        return Objects.equals(id, admin.id);
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(id);
+    }
```

Comment:
Just for clean code's sake, imo those methods should come last behind every other method. QA is to be asked about this issue.